README.md

```diff
@@ -10,6 +10,10 @@ SYNOPSIS
     easy-stage-dataset -t <submission-timestamp> -u <urn> -d <doi> [ -o ] \
                           <EASY-bag> <staged-digital-object-set>
 
+    easy-stage-file-item [<options>...] <staged-digital-object-set>
+    easy-stage-file-item <staged-digital-object-set> <csv-file>
```

Comment:
I see your point. How about making csv-file a named option? So

     easy-stage-file-item [<options>...] <staged-digital-object-set>

would be the only command line (for file items) and there'd be an option, say `--input-file`, to indicate the csv file. It would of course be mutually exclusive with the other options.